README.md

```diff
@@ -37,6 +37,7 @@ Cargo requires the following tools and packages to build:
 * `cmake`
 * `pkg-config`
 * OpenSSL headers (`libssl-dev` package on ubuntu)
+* libgit2
```

Comment:
Currently libgit2 is built from source if it's not found on the system, so this shouldn't be necessary as a build prerequisite.